test/Chirp.Infrastructure.Tests/RestrictedCheepsUnitTests.cs

```diff
@@ -0,0 +1,105 @@
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using FluentAssertions;
+
+public class RestrictedCheepTests
+{
+    private readonly SqliteConnection? _connection; //Connection to the database
+    private readonly ChirpDBContext _context; //Context for the database
+    private readonly CheepRepository repository; //Repository for the database
+
+    public RestrictedCheepTests()
+    {
+        //Arrange
+        //Creates a database in memory - Makkes connection string before opening the connection
+        var builder = new DbContextOptionsBuilder<ChirpDBContext>();
+        builder.UseSqlite("Filename=:memory:");
+        ChirpDBContext context = new(builder.Options);
+        _connection = context.Database.GetDbConnection() as SqliteConnection;
+        if (_connection != null)  //Takes care of the null exception
+        {
+            _connection.Open();
+        }
+        context.Database.EnsureCreated();
+
+        //Creates a cheep and author to add to the database
```

Comment:
I suggest that we move this out, into each test, because, in my opinion, this makes the test cases less readable and more prone to erros, if more test were to be added, or existings tests were to be changed.
I myself, used som time figuring out what exactly was tested in the test, since the data is added in the constructor already.

Reply:
The 4 tests in that class, are using the same objects and so on, so it would be redundant to have it in each test, since the only thing changed for each of the four, is the message itself. 
It would also make the whole script almost twice as big, i can try adding comments that make it more readable? Which describes what the constructor does in the beginning of each test?